packages/docs/docs/about.mdx

```diff
@@ -88,4 +88,11 @@ function CharacterGenerator() {
 }
 ```
 
-[![AI.JSX demo](https://asciinema.org/a/kLwqpHVf2jsOU0vxEpl8f47zw.svg)](https://asciinema.org/a/kLwqpHVf2jsOU0vxEpl8f47zw)
+import AsciinemaWidget from '../src/components/AsciinemaWidget';
```

Comment:
Not sure if I should put this at the top or not.
Probably this is easier to copy&paste later on.

Reply:
This is fine.